drivers/gles3/storage/material_storage.cpp

```diff
@@ -1517,6 +1517,8 @@ MaterialStorage::MaterialStorage() {
 		actions.renames["VIEW_MONO_LEFT"] = "0";
 		actions.renames["VIEW_RIGHT"] = "1";
 
+		actions.renames["POLAR_COORDINATES"] = "polar_coordinates";
```

Comment:
The other exposed builtin functions are all lowercase, so this should be too.

Reply:
@clayjohn Thanks for noticing! I updated the exposed function to lowercase. Somehow though when the shader compilation runs, the engine crashes (apparently due to some overloading error, it seems)

```bash
ERROR: 0:2082: 'polar_coordinates' : no matching overloaded function found
ERROR: 0:2082: 'assign' :  cannot convert from ' const float' to ' temp highp 2-component vector of float'
ERROR: 0:2082: '' : compilation terminated
```